circle/stop-competitors.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sudo service mysql stop
+sudo service postgresql stop
+sudo service mongodb stop
```

Comment:
yeah, we're  not using any of these things that circle "helpfully" starts up for us (even the services that we do use, we start our own docker'ed versions of)